CONTRIBUTING.md

```diff
@@ -124,6 +124,12 @@ This will also test that the documentation build passes and run the style checks
 $ tox -e py
 ```
 
+If you want to run the integration tests in your current python environment, you can do:
+
+```bash
+pytest -m "integration"
```

Comment:
This is the benefit of marking the tests as integration

Reply:
We might want to add that to run integration tests "Set environment variables C42_USERNAME and C42_PW with CCA credentials".

Reply:
I added this information in `py42` repo but somehow missed it here.